package.json

```diff
@@ -1,4 +1,6 @@
 {
```

Comment:
can you change this line to fix th css build:

`"build": "node-sass --include-path node_modules static/sass --source-map true --output-style compressed --output static/css && postcss --map false --use autoprefixer --replace 'static/css/**/*.css'",`